### ISSUE TITLE
train a new model

### ISSUE THREAD

**Issue by biocc** [Noble-Lab/casanovo#114]
![image](https://user-images.githubusercontent.com/77394889/211506292-0f451f69-3e25-4274-8fe8-9f58bb137b68.png)
[casanovo_20230105052124.log](https://github.com/Noble-Lab/casanovo/files/10380858/casanovo_20230105052124.log)
Dear sir,
I want to train a new model. After i ran the casanovo, there were more than 30 model files (.cpkt) output. I don't know which one is the best model in the end. Or there are something wrong during my excuting.

**Comment by bittremieux** [comment]
By default, Casanovo makes a snapshot of the model weights every 50,000 training steps. You can optionally [modify this in the config file](https://github.com/Noble-Lab/casanovo/blob/main/casanovo/config.yaml#L79), depending on your dataset size. 

To select the best performing model, you can check the training loss, validation loss, amino acid precision, and peptide precision in the log file produced by Casanovo. From the attached log, it seems that the model after 550,000 achieved the best performance.

Side note: Both your training and validation loss went up again after 550,000 training steps. I suggest doing another training run with a lower learning rate to see if that improves the performance.
